source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -0,0 +1,83 @@
+}
+
+<div class="content-evaluation evaluation">
+    <h2 class="title-evaluations">Devolución de <strong>Making Sense</strong> a <strong data-bind="text: evaluation.fullName()"></strong></h2>
+    <div class="top-evaluation">
+        <div class="top-left">
+            <h4>Información Personal</h4>
+            <ul class="personal-info-container clearfix">
+                <li><span class="item">Puesto:</span><span class="item-info" data-bind="text: evaluation.currentPosition()"></span></li>
+                <li><span class="item">Evaluadores:</span><span class="item-info" data-bind="text: evaluation.fullName()"></span></li>
+                <li><span class="item">Seniority:</span><span class="item-info" data-bind="text: evaluation.seniority()"></span></li>
+                <li><span class="item">Responsable:</span><span class="item-info" data-bind="text: evaluation.responsibleId()"></span></li>
+                <li><span class="item">Fecha:</span><span class="item-info">18/07/2015</span></li>
+                <li>
+                    <span class="item">Proyectos:</span>
+                    <div class="editable-projects" data-bind="css: { edition: evaluation.project()}">
+                        <input type="text" placeholder="Ingrese los proyectos" data-bind="value: evaluation.project"/><span class="pencil"></span>
+                    </div>
+                </li>
+            </ul>
+            <h4>Métricas de Evaluación</h4>
+            <div class="metrics-container">
+                <span>1</span> No alcanza las Expectativas - <span>2</span> Necesita Mejorar<br/>
+                <span>3</span> Cumple con Expectativas - <span>4</span> Excede Expectativas
+            </div>
+            <p class="explanatory-text">*Si no tenes conocimientos para calificar un campo puedes dejarlo vacio</p>
+        </div>
+        <div class="top-right">
+            <h4>Comentarios Generales</h4>
+            <textarea class="comentarios-generales" placeholder="Dejá tu comentario aquí..."></textarea>
+            <button class="button-evaluations comment-btn">Comentar</button>
+        </div>
+    </div>
+    <table class="table calification-items table-3-columns hide-column-2">
+        <thead>
+            <tr>
+                <th class="calificators"></th>
+                <th class="calificators"><i class="icon top-tip" data-tips="" data-calificator-col="1"></i>Auto Evaluacion</th>
+                <th class="calificators"><i class="icon top-tip" data-tips="" data-calificator-col="2"></i>Calentey</th>
+                <th class="calificators">Empresa</th>
+            </tr>
+            <tr>
+                <th class="title-evaluation">Calificación Final</th>
+                <th class="title-evaluation"><span>3.5</span></th>
+                <th class="title-evaluation"><span>4</span></th>
+                <th class="title-evaluation">4</th>
+            </tr>
+            <tr>
```

Comment:
I understand that it is hard-coded by the moment, but then it will be inside `foreach`, right?